app/src/main/java/com/example/yard/HelperClass/QueryPosts.java

```diff
@@ -0,0 +1,32 @@
+package com.example.yard.HelperClass;
+import com.example.yard.Adapter.PostCreation;
+import com.example.yard.Adapter.PostsAdapter;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.List;
+
+public class QueryPosts {
+  public void queryPosts(boolean isFromBio, PostsAdapter adapter, List<PostCreation> allPosts) {
```

Comment:
this can be static
Please google and read more about static functions in Java

`public static void queryPosts(`